assignments/arrays.js

```diff
@@ -64,33 +64,54 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
 console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
-
+let id33 = [];
+for(let i = 0; i < inventory.length ; i++){
+    id33.push(inventory[i])
+}
 
```

Comment:
Hey Kevin, I want you to take a look at the solution code for this. It should look a little more of something like 
console.log(`Car 33 is a ${inventory[32].car_year} ${inventory[32].car_make} ${inventory[32].car_model}` );

There are still other ways of doing this, but I think this may be the best